src/content/docs/en/guides/integrations-guide/svelte.mdx

```diff
@@ -103,44 +103,28 @@ To use your first Svelte component in Astro, head to our [UI framework documenta
 
 This integration is powered by `@sveltejs/vite-plugin-svelte`. To customize the Svelte compiler, options can be provided to the integration. See the [`@sveltejs/vite-plugin-svelte` docs](https://github.com/sveltejs/vite-plugin-svelte/blob/HEAD/docs/config.md) for more details.
 
-### Default options
+You can set options either by passing them to the `svelte` integration in `astro.config.mjs` or in `svelte.config.js`. The options in `astro.config.mjs` will take precedence over the options in `svelte.config.js` if both are present:
 
-This integration passes the following default options to the Svelte compiler:
-
-```js
-const defaultOptions = {
-  emitCss: true,
-  compilerOptions: { dev: isDev, hydratable: true },
-  preprocess: vitePreprocess(),
-};
-```
-
-These `emitCss`, `compilerOptions.dev`, and `compilerOptions.hydratable` values are required to build properly for Astro and cannot be overridden.
-
-Providing your own `preprocess` options **will** override the [`vitePreprocess()`](https://github.com/sveltejs/vite-plugin-svelte/blob/HEAD/docs/preprocess.md) default. Make sure to enable the preprocessor flags needed for your project.
-
-You can set options either by passing them to the `svelte` integration in `astro.config.mjs` or in `svelte.config.js`. Either of these would override the default `preprocess` setting:
-
-```js title="astro.config.mjs" "preprocess: []"
+```js title="astro.config.mjs" "extensions: ['.svelte']"
 import { defineConfig } from 'astro/config';
 import svelte from '@astrojs/svelte';
 
 export default defineConfig({
-  integrations: [svelte({ preprocess: [] })],
+  integrations: [svelte({ extensions: ['.svelte'] })],
 });
 ```
 
 ```js title="svelte.config.js"
 export default {
-  preprocess: [],
+  extensions: ['.svelte'],
 };
 ```
 
-## Intellisense for TypeScript
+## Preprocessors
 
 <Since v="2.0.0" pkg="@astrojs/svelte" />
 
-If you're using a preprocessor like TypeScript or SCSS in your Svelte files, you can create a `svelte.config.js` file so that the Svelte IDE extension can correctly parse the Svelte files.
+If you're using SCSS or Stylus in your Svelte files, you can create a `svelte.config.js` file so that they are preprocessed by Svelte, and the Svelte IDE extension can correctly parse the Svelte files.
```

Comment:
I removed references to TypeScript here as Svelte 5 now handles TypeScript by default even without `vitePreprocess`.